recipes/vectorbt/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "vectorbt" %}
+{% set version = "0.26.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b46e826ca506b495a77c9223542f0eb0adac3239ae61846f57664bebda91880a
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - numpy >=1.16.5, <2.0.0
```

This comes from [`setup.py` upstream](https://github.com/polakowo/vectorbt/blob/f079b27a59e21a125d5c622da9b7be4160f939e6/setup.py#L24) but this is improperly formatted for package manager like `mamba`. The following is better:
```suggestion
    - numpy >=1.16.5,<2.0.0
```